new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -204,134 +231,398 @@ public static void initAll(@Namespaces(6) List<String> namespaces) {
-        "weblogic_servlet_invocation_total_count%7Bapp%3D%22wlsexporter%22%7D%5B15s%5D";
-    checkMetricsViaPrometheus(testappPrometheusSearchKey, "wlsexporter");
+    verifyMonExpAppAccessThroughNginx(ingressHost1List.get(0),1);
+
+    try {
+      logger.info("Testing replace configuration");
+      replaceConfiguration();
+      logger.info("Testing append configuration");
+      appendConfiguration();
+      logger.info("Testing replace One Attribute Value AsArray configuration");
+      replaceOneAttributeValueAsArrayConfiguration();
+      logger.info("Testing append One Attribute Value AsArray configuration");
+      appendArrayWithOneExistedAndOneDifferentAttributeValueAsArrayConfiguration();
+      logger.info("Testing replace with empty configuration");
+      replaceWithEmptyConfiguration();
+      logger.info("Testing append with empty configuration");
+      appendWithEmptyConfiguration();
+      logger.info("Testing append with invalid yaml configuration");
+      appendWithNotYmlConfiguration();
+      logger.info("Testing replace with invalid yaml configuration");
+      replaceWithNotYmlConfiguration();
+      logger.info("Testing append with corrupted yaml configuration");
+      appendWithCorruptedYmlConfiguration();
+      logger.info("Testing replace with corrupted yaml configuration");
+      replaceWithCorruptedYmlConfiguration();
+      logger.info("Testing replace with dublicated values yaml configuration");
+      replaceWithDublicatedValuesConfiguration();
+      logger.info("Testing append with corrupted yaml configuration");
+      appendWithDuplicatedValuesConfiguration();
+      logger.info("Testing replace with name snake false yaml configuration");
+      replaceMetricsNameSnakeCaseFalseConfiguration();
+      logger.info("Testing change with no credentials configuration");
+      changeConfigNoCredentials();
+      logger.info("Testing change with no invalid user configuration");
+      changeConfigInvalidUser();
+      logger.info("Testing change with no invalid pass configuration");
+      changeConfigInvalidPass();
+      logger.info("Testing change with empty user configuration");
+      changeConfigEmptyUser();
+      logger.info("Testing change with no empty pass configuration");
+      changeConfigEmptyPass();
+      logger.info("Testing replace with domain qualifier configuration");
+      replaceMetricsDomainQualifierTrueConfiguration();
+    } finally {
+      //restore configuration
+      submitConfigureForm(exporterUrl, "replace", RESOURCE_DIR + "/exporter/exporter-config.yml");
+    }
   }
 
   /**
-   * Install Prometheus, Grafana using helm chart, and verify that pods are running.
-   * @throws ApiException when creating helm charts or pods fails
+   * Test covers the following use cases.
+   * Create Prometheus, Grafana from latest version of helm chart
+   * verify access to monitoring exporter WebLogic metrics via nginx
+   * check WebLogic metrics via Prometheus
```

Comment:
Fix the javadoc. Similar to Sankar's comment earlier.

Reply:
done